models/predefined_exporters.go

```diff
@@ -71,24 +76,57 @@ func IsErrPredefinedExporterNotFound(e error) bool {
 	return ok
 }
 
-func FromPredefinedExporter(predefinedExporter, promNetwork string, exported types.ContainerJSON) (Exporter, error) {
+func FromPredefinedExporter(predefinedExporter string, exported types.ContainerJSON) (Exporter, error) {
 	p, ok := predefinedExporters[predefinedExporter]
 	if !ok {
 		return Exporter{}, newErrPredefinedExporterNotFound(predefinedExporter)
 	}
 
-	cmd := make([]string, 0)
+	cmd, err := renderSliceOfTpls(p.cmd, exported)
+	if err != nil {
+		return Exporter{}, err
+	}
+
+	envVars, err := renderSliceOfTpls(p.envVars, exported)
+	if err != nil {
+		return Exporter{}, err
+	}
+
+	return NewExporter(predefinedExporter, p.image, cmd, envVars, exported), nil
+}
+
+func renderSliceOfTpls(tpls []string, values interface{}) ([]string, error) {
```

Comment:
Can you add a doc block for the return of this function ? What does the `cmd []string` return represent ?
Also, I'm not sure to get the logic here : this function renders parts of templates by calling a `renderTpl` function, so what's the final aim ? Do you intend to render multiple files, or many parts to join them next ?